elements/camera-controls.html

```diff
@@ -0,0 +1,170 @@
+                color: white;
+            }
+        </style>
+
+        <iron-ajax
+            id="ajaxMoveCamera"
+            method="POST"
+            handle-as="json"
+            content-type="application/json"
+            on-response="onMoveCamera"></iron-ajax>
+
+        <iron-ajax
+            id="ajaxStopCamera"
+            on-response="onStopCamera"></iron-ajax>
+
+        <iron-ajax
+            id="ajaxSavePreset"
+            method="POST"
+            handle-as="json"
+            content-type="application/json"
+            on-response="onSavePreset"></iron-ajax>
+
+        <iron-ajax
+            id="ajaxRecallPreset"
+            method="POST"
+            handle-as="json"
+            content-type="application/json"
+            on-response="onRecallPreset"></iron-ajax>
+
+        <div class="layout horizontal">
+            <div class="layout horizontal">
+                <div class="flex">
+                    <paper-button on-down="_moveCameraLeft" on-up="stopCamera">left</paper-button>
+                    <div class="layout vertical">
+                        <paper-button on-down="_moveCameraUp" on-up="stopCamera">up</paper-button>
+                        <paper-button on-down="_moveCameraDown" on-up="stopCamera">down</paper-button>   
+                    </div>
+                    <paper-button on-down="_moveCameraRight" on-up="stopCamera">right</paper-button>
+                </div>
+            </div>
+            <div class="flex"></div>
+            <div class="layout vertical">
+                <template is="dom-repeat" items="{{presets}}">
+                    <div>
+                        <paper-button id="{{item}}" on-tap="savePreset">savepreset {{item}}</paper-button>
+                        <paper-button id="{{item}}" on-tap="recallPreset">recall preset {{item}}</paper-button>
+                    </div>
+                </template>
+            </div>
+        </div>
+    </template>
+
+    <script>
+        (function() {
+            'use strict';
+            class cameraControls {
+                beforeRegister() {
+                    this.is = 'camera-controls';
+
+                    this.properties = {
+                        presets: {
+                            type: Object,
+                            value: function() {
+                                var list = [];
+                                for (var i = 1; i <= 99; i++) {
+                                    list.push(i);
+                                }
+                                return list;
+                            }
+                        }
+                    };
+                }
+
+                stopCamera() {
+                    this.$.ajaxStopCamera.url = "cameras/stop";
+                    this.$.ajaxStopCamera.generateRequest();
+                }
+
+                onStopCamera(e, request) {
+                    if (request.succeeded) {
+                        console.log(request.message);
```

Comment:
This is to indicate we need to handle errors. Currently the frontend is only a mock, implementation will come later.